build.gradle

```diff
@@ -3,7 +3,6 @@ import se.bjurr.gitchangelog.plugin.gradle.GitChangelogTask
 buildscript {
 	repositories {
 		maven { url = "https://files.minecraftforge.net/maven" }
-		jcenter()
```

Comment:
I don't think this should be changed

Reply:
I'll address the other comments in a bit, this change should stay though: [jcenter won't be available for much longer](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/), and since it isn't actually used I think removing it now is better than getting build failures when jcenter actually shuts down.